### ISSUE TITLE
README: consume and replaygain

### ISSUE THREAD

**Issue by markreg** [ejurgensen/forked-daapd#383]
consume and replaygain are honored by forked-daapd. Should these be removed from the README since they do work?

**Comment by ejurgensen** [comment]
@chme, I think this is in the mpd section of the README - what do you say?

**Comment by chme** [comment]
consume is supported by forked-daapd and the README should be adapted. The informations for ympd and mpc probably also need an update. I can have a look at that.

Regarding replaygain, at least the mpd commands are not supported (setting/getting replay gain mode). I am not sure if replaygain informations in the music files are honored by forked-daapd (i can vaguely remember that you experimented with some ffmepg options?)

**Comment by ejurgensen** [comment]
Yes, correct, I once experimented with including a ffmpeg replay-gain filter, but I couldn't find out if it worked, so I never included it. The user requesting the feature didn't test it, and it wasn't really something I needed myself.

So if some replaygain is working it is not thanks to me :-) Maybe it is something that some versions of ffmpeg do automatically?
